bin/plugin/commands/build-plugins.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs-extra' );
+const path = require( 'path' );
+const glob = require( 'fast-glob' );
+
+/**
+ * Internal dependencies
+ */
+const { log, formats } = require( '../lib/logger' );
+const { getModuleDataFromHeader, getModuleHeader } = require( './common' );
+
+exports.options = [];
+
+/**
+ * Command for copying the files required for the standalone plugin.
+ */
+exports.handler = async () => {
+	const pluginsFile = path.join( '.', 'plugins.json' );
+	fs.readFile( pluginsFile, 'utf8', ( err, jsonString ) => {
+		if ( err ) {
+			log(
+				formats.error(
+					`Error reading file from disk: "${ err }"`
+				)
+			);
+		}
+
+		try {
+			const plugins = JSON.parse( jsonString );
+			for ( const moduleDir in plugins ) {
+				const pluginVersion = plugins[ moduleDir ]?.version;
+				const pluginSlug = plugins[ moduleDir ]?.slug;
+
+				if ( ! pluginVersion || ! pluginSlug ) {
+					log(
+						formats.error(
+							'The given module configuration is invalid, the module JSON object is missing the "version" and/or "slug" properties, or they are misspelled.'
+						)
+					);
+					return;
+				}
+
+				try {
+					// Copy module files from the folder.
+					const modulePath = path.join( '.', 'modules/' + moduleDir );
+					const buildModulePath = path.join( '.', 'build/' + pluginSlug );
+					try {
+						// Clean up build module files directory.
+						fs.rmSync( buildModulePath, { force: true, recursive: true } );
+
+						fs.copySync( modulePath, buildModulePath, { overwrite: true } );
+					} catch ( copyError ) {
+						log(
+							formats.error(
+								`Error copying plugin file: "${ copyError }"`
+							)
+						);
+						continue;
+					}
+
+					// Update file content.
+					updatePluginHeader( {
+						originalModulePath: moduleDir,
```

Comment:
```suggestion
						modulePath: moduleDir,
```

More of an observation that we probably don't need to call this `originalModulePath` and can just call it `modulePath` because the new path is the plugin path, the module path will always remain as it currently is.

Not a blocking change, more of an observation.